docs/api.md

```diff
@@ -372,6 +372,14 @@ sharp('input.png')
   });
 ```
 
+#### maskWithColor(rgba)
```

Comment:
These details should move to the `withOverlay` method. I'm happy to fix-up the docs if you're unsure about this.